src/og/utils/shared.js

```diff
@@ -332,12 +332,12 @@ export function binaryInsert(ar, el, compare_fn) {
 /**
  * Returns two segment lines intersection coordinate.
  * @static
- * @param {og.math.Vec2} start1 - First line first coordinate.
- * @param {og.math.Vec2} end1 - First line second coordinate.
- * @param {og.math.Vec2} start2 - Second line first coordinate.
- * @param {og.math.Vec2} end2 - Second line second coordinate.
+ * @param {math.Vec2} start1 - First line first coordinate.

```

Comment:
When we will use this module lets remove math from Vec2. In future. Right now it is ok:)